tests/e2e/e2e_ibc_test.go

```diff
@@ -64,17 +75,13 @@ func (s *E2ETest) TestIBCTokenTransfer() {
 	// because we won't have price for it.
 	s.Run("send_stake_to_umee", func() {
 		// require the recipient account receives the IBC tokens (IBC packets ACKd)
-		stakeIBCHash := "ibc/C053D637CCA2A2BA030E2C5EE1B28A16F71CCB0E45E8BE52766DC1B241B77878"
 		umeeAPIEndpoint := s.UmeeREST()
-
-		valAddr, err := s.Chain.Validators[0].KeyInfo.GetAddress()
-		s.Require().NoError(err)
-		recipient := valAddr.String()
+		recipient := s.AccountAddr(0).String()
 
 		token := sdk.NewInt64Coin("stake", 3300000000) // 3300stake
-		s.SendIBC(setup.GaiaChainID, s.Chain.ID, recipient, token, false)
-
-		s.checkSupply(umeeAPIEndpoint, stakeIBCHash, token.Amount)
+		s.SendIBC(setup.GaiaChainID, s.Chain.ID, recipient, token, false, "")
+		// Zero, since not a registered token
+		s.checkSupply(umeeAPIEndpoint, stakeIBCHash, sdk.ZeroInt())
```

Comment:
This should not have been nonzero (see comment), but ti was working before for some reason.